db/migrate/20190703155941_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
```

Comment:
maybe add email, password and index them uniquely by email

Reply:
I prefer to wait for when installing devise, ok ?

Reply:
I know that devise has built in generators for that

Reply:
makes sense